OrbitCore/SamplingProfiler.cpp

```diff
@@ -168,7 +202,7 @@ void SamplingProfiler::ProcessSamples() {
   for (auto& dataIt : m_ThreadSampleData) {
     ThreadSampleData& threadSampleData = dataIt.second;
 
-    threadSampleData.ComputeAverageThreadUsage();
+   ComputeAverageThreadUsage(&threadSampleData);
```

Comment:
clang-format? indentation does not look right...

Reply:
Fixed now